### DISCUSSION TITLE
Docs suggestion: mention bot limitations in a separate page

### DISCUSSION THREAD

**Discussion by HonkingGoose** [renovatebot/renovate#11884] [Suggest an Idea]
I've seen questions that go something like this:

- "Why doesn't Renovate bot automerge right away, while I keep pushing commits to the target branch?"
- "Why can't Renovate bot automerge 20 grouped dependencies at once, instead of doing them in pairs?"
- "Why doesn't Renovate react right away when I click the "rebase this PR" checkbox?"

From this it seems that people expect Renovate to be able to process their repository in mere seconds. I think this is partly because we do not set the proper expectations up front, so people are free to imagine a super-fast bot. 😄 

I propose we create a new page where we explain the performance limits of the bot. I'm not sure myself of the limits of the bot, so I cannot write the docs in full. I'm wiling to help write the pages of course, and I will review whatever the maintainers write up. 😉

**Comment by rarkins** [comment]
Sounds good. I guess it's like "Known limitations" with an explanation of whether that's a permanent design decision or something which is aiming to be improved or fixed along with a relevant issue number to track.

I would prefer that these limitations have as few manager-specific details as possible because those would be better put in the respective manager or datasource module docs.

**Reply by HonkingGoose** [reply]
> Sounds good. I guess it's like "Known limitations" with an explanation of whether that's a permanent design decision or something which is aiming to be improved or fixed along with a relevant issue number to track.

Yeah, that's basically what I'm getting at.

> I would prefer that these limitations have as few manager-specific details as possible because those would be better put in the respective manager or datasource module docs.

Agree, a general overview should be enough to set the proper expectations of the general performance of the bot. If manager specific stuff is broken/slow, it should be documented in their docs, not in the general overview.

**Comment by HonkingGoose** [comment]
Resolved by PR: https://github.com/renovatebot/renovate/pull/11946
